wgpu/src/backend/direct.rs

```diff
@@ -392,6 +392,13 @@ fn map_texture_tagged_copy_view(
     }
 }
 
+fn map_store_op(op: StoreOp) -> wgc::command::StoreOp {
+    match op {
+        StoreOp::Store => wgc::command::StoreOp::Store,
+        StoreOp::Discard => wgc::command::StoreOp::Discard,
+    }
+}
```

Comment:
We totally should lift this into wgt and use the same type everywhere

Reply:
I was about to, but then figured since the LoadOp can't go into wgt (it has the clear value with it for improved Rust ergonomics) I didn't want to put them into different places so it's easier to find stuff 🤔

Reply:
Fair